ecology.yaml

```diff
@@ -0,0 +1,525 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: 'Statistics'
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: 'Statistics'
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: 'Annotation'
+
+  - name: add_input_name_as_column
+    owner: mvdbeek
+    tool_panel_section_label: 'Text Manipulation'
+ 
+  - name: regex_replace
```

Comment:
This one can probably be replaces by other tools already installed and is probably very old isn't it?

Reply:
In fact this tool is already in [usegalaxy.eu](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/kellrott/regex_replace/regex_replace/1.0.0). So maybe it can be moved towards the "trusted" section ?

Reply:
Thank you! So I add the kellrott regex repo as trusted